src/infection-model/matrixUtils.tsx

```diff
@@ -0,0 +1,31 @@
+// helpers for the ndarray package
```

Comment:
I don't think you need this line -- it's clear that you need it below. Let me know if I'm misunderstanding?